helmfile.d/charts/gatekeeper/podsecuritypolicies/templates/_.tpl

```diff
@@ -41,6 +41,12 @@ kinds:
 namespaces:
   - {{ .namespace }}
 labelSelector:
+  {{- if .labels }}
   matchLabels:
     {{- toYaml .labels | nindent 4 }}
+  {{- end }}
+  {{- if .expressions }}
+  matchExpressions:
+    {{- toYaml .expressions | nindent 4 }}
+  {{- end }}
```

Comment:
Do we want to be able set matchLabels and matchExpressions, or should it only be possible to specify either (thinking if it should be if-elif instead of if-if)?

Reply:
Could there be a problem allowing both?

Reply:
I don't think there will be a problem allowing both, atleast according to my tests.

Reply:
I will resolve this now then.